src/app/company/models/company.model.ts

```diff
@@ -0,0 +1,20 @@
+export class CompanyItem {
+    id: number;
+    userId: number;
+    companyName: string;
+    businessId: string;
+    street: string;
+    streetNumber: string;
+    city: string;
+    zipCode: string;
+}
+
+export class AddCompanyItem {
+    userId: number;
```

Comment:
Len sa zamýšľam - Potrebujeme v tejto entite mať userId? Ak to aj api vyžaduje, prečítame si to v rámci efektu zo store (tam je to jednoduchšie), nemusíme túto informáciu dodatočne čítať zo store v komponente...
Môže sa potom chybou stať, že pridám company inému užívateľovi, než je aktuálne prihlásený?...